website/content/en/docs/building-operators/helm/reference/advanced_features/reconcile-period.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Reconcile Period from Custom Resource Annotations
+linkTitle: Custom Resource Annotations Reconcile Period
+weight: 200
+description: Allow a user to set the desired reconcile period from the custom resource's annotations
+---
+
+While running a helm operator, we can specify the reconcile-period through the custom resource's annotations under the `helm.operator-sdk/reconcile-period` key. Using this feature will reconcile the cluster in a fixed interval of time. The reconcile period can be specified in the custom resource's annotations in the following manner: 
```

Comment:
Also, does having a reconcile-period actually mean it has a fixed interval reconcile? I thought the reconcile periods were a MINIMUM meaning if no reconcile has happened then run a reconcile. If events come in before that period will the operator not reconcile?

Reply:
Thanks for calling that out. You're right - this ensures that at a minimum the cluster will reconcile in the specified interval of time. However, if events come in between before the period elapses, the cluster **will** get reconciled (as it should). This feature is to "ensure" that the cluster does not go long periods of time without reconciling. 

I've updated the description in the docs with that explanation.